src/plugins/gui/fiatProviderTypes.js

```diff
@@ -61,7 +61,8 @@ export type FiatProvider = {
 export type FiatProviderFactoryParams = {
   io: {
     store: Object
-  }
+  },
+  apiKeys?: Object | string
```

Comment:
I think this should be `unknown`, and then expect the plugins to use `asString` / `asObject({ something: asString })` to validate them.

Reply:
Ok. That's fair.